### DISCUSSION TITLE
Bucket Mode - Mekanism

### DISCUSSION THREAD

**Discussion by maccrushington** [NillerMedDild/Enigmatica6#3812] [Q&A] [answered]
Cannot seem to get bucket mode working, cannot seem to find a key bind in controls.  Is it missing or am I missing something, used to be a sneak right click, but that does nothing.  thanks.

**Comment by theboo** [comment, marked as answer]
Try sneak scrollwheel

**Reply by maccrushington** [reply]
that worked... sneak, scroll... for a moment my brain was going sneak-middle click... because I am a derp.  but yea, got it working now.  Thanks a ton ♥
